ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -231,44 +231,44 @@ public void processVRInput(RDXVRContext vrContext)
-         if (aButton.bChanged() && !aButton.bState())
-         {
-            streamToController.set(!streamToController.get());
-         }
-
-         // NOTE: Implement hand open close for controller trigger button.
-         InputDigitalActionData clickTriggerButton = controller.getClickTriggerActionData();
-         if (clickTriggerButton.bChanged() && !clickTriggerButton.bState())
-         {
-            HandConfiguration handConfiguration = nextHandConfiguration(RobotSide.LEFT);
-            sendHandCommand(RobotSide.LEFT, handConfiguration);
-         }
-
-         // Check if left joystick is pressed in order to trigger recording or replay of motion
-         InputDigitalActionData joystickButton = controller.getJoystickPressActionData();
-         kinematicsRecorder.processRecordReplayInput(joystickButton);
-         if (kinematicsRecorder.isReplayingEnabled().get())
-            wakeUpToolbox();
-      });
+                                                             {
+                                                                InputDigitalActionData aButton = controller.getAButtonActionData();
+                                                                if (aButton.bChanged() && !aButton.bState())
+                                                                {
+                                                                   streamToController.set(!streamToController.get());
+                                                                }
+
+                                                                // NOTE: Implement hand open close for controller trigger button.
+                                                                InputDigitalActionData clickTriggerButton = controller.getClickTriggerActionData();
+                                                                if (clickTriggerButton.bChanged() && !clickTriggerButton.bState())
+                                                                {
+                                                                   HandConfiguration handConfiguration = nextHandConfiguration(RobotSide.LEFT);
+                                                                   sendHandCommand(RobotSide.LEFT, handConfiguration);
+                                                                }
+
+                                                                // Check if left joystick is pressed in order to trigger recording or replay of motion
+                                                                InputDigitalActionData joystickButton = controller.getJoystickPressActionData();
+                                                                kinematicsRecorder.processRecordReplayInput(joystickButton);
+                                                                if (kinematicsRecorder.isReplayingEnabled().get())
+                                                                   wakeUpToolbox();
+                                                             });
```

Comment:
undo indents (using middle click drag to block select the space and hit the delete key is the easiest way to do that)